frontend/src/containers/Dashboard/reducers.js

```diff
@@ -0,0 +1,32 @@
+import {
+  HOUSE_ISSUES_FETCH_DATA_SUCCESS,
+  HOUSE_ISSUES_IS_LOADING,
+  HOUSE_ISSUES_HAS_ERRORED
+} from './actions';
+
+export function houseIssuesHasErrored( state = false, action) {
```

Comment:
In my opinion this should be one reducer. I.e. which stores all the booleans and housing issues. Rather than three separate ones. I.e. see the login/logout one. Similar things grouped together. I would think this is slightly too highly abstracted.

Reply:
ok again I don't really know enough to have an opinion. Will revisit this one as well.

Reply:
ah @Mortuie  one reducer with three cases in the switch statement? so one object for state with three members. I'll give it a go

Reply:
@Mortuie default state will have to be bool or array? or two reducers, one for bools and one for empty array?

Reply:
oh @Mortuie  ignore last comment, defaults will be inside the object, I'll have another go.